src/DataFixtures/AppFixtures 2.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\DataFixtures;
+
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Common\Persistence\ObjectManager;
+use App\Entity\Academy;
+
+class AppFixtures extends Fixture
+{
+    public function load(ObjectManager $manager)
+    {
+        for ($i = 0; $i < 8; $i++) {
+            $academy = new Academy();
+
+            $academy->setAcademyName('Super Academy '.$i);
+            $academy->setAcademyEmail('academy@email.com');
+            $academy->setAcademyUrl('www.academy.com');
+            $academy->setAcademyLogo('www.google.com/image.jpg');
+            $manager->persist($academy);
+        }
+        $manager->flush();
+    }
+}
```

Comment:
Is this really needed? same file in master :thinking:

Reply:
Creating fixtures by the book:
https://symfony.com/doc/master/bundles/DoctrineFixturesBundle/index.html

Reply:
Palauk,Fixtures 2??? Tuoj pachekinsiu